src/stt/src/SteinerTreeBuilder.cpp

```diff
@@ -120,7 +122,7 @@ Tree SteinerTreeBuilder::makeTree(std::vector<int>& x,
     int y_arr[pin_count];
 
     std::copy(x.begin(), x.end(), x_arr);
-    std::copy(y.begin(), y.end(), x_arr);
+    std::copy(y.begin(), y.end(), y_arr);
```

Comment:
@maliberty is it recommended/safe to use x.data() and y.data() instead of copying the vectors?

Reply:
It would work but better still would be to make flute take a vector& and just pass it through.